### PR TITLE
Call out the Sequel::Dataset#all optimization in the Sinatra example

### PR DESCRIPTION
**Pull request by mwpastore**
* Also remove an old item from "Unfinished business"; support for explicit serializer discovery was added in v0.5.0. :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -575,15 +582,13 @@ class BaseSerializer
 end
 
 class PostSerializer < BaseSerializer
-  attribute :title
-  attribute :content
+  attribute :title, :content
```
Comment (fotinakis) [inline README.md]: You have to use `attributes` to pass multi attributes. Here and below.

**Comment by fotinakis** [comment]
Back to you @mwpastore

File: README.md (hunk 1 above)
Comment (mwpastore) [inline README.md]: Darn, nice catch!

**Comment by fotinakis** [comment]
Thanks!

**Pull request merged**